### PR TITLE
92 branch

### PR DESCRIPTION
**Pull request by ukenhk-kuni077**
ADD category , location listbox to task detail.

### CODE REVIEW

File: src/Task.vue [hunk 1]
```diff
@@ -9,6 +9,19 @@
...
+        date : <input type="date" v-model.sync="task.date"></input>
+        location : <select v-model.sync="task.location">
+          <option>スーパー</option>
+          <option>駅</option>
+          <option>自宅</option>
+        </select>
+        category : <select v-model.sync="task.category">
+          <option>shopping</option>
+          <option>visit</option>
+          <option>other</option>
```
Comment (amanoese) [inline src/Task.vue]: need attribute "value"

Comment (amanoese) [inline src/Task.vue]: need attribute "value"

File: src/Task.vue [hunk 2]
```diff
@@ -9,6 +9,19 @@
         {{ task.title }}
       </h5>
       <p>{{ task.body }}</p>
+      <p>
+        date : <input type="date" v-model.sync="task.date"></input>
+        location : <select v-model.sync="task.location">
```
Comment (amanoese) [inline src/Task.vue]: select tag need label tag.

matteriarize is restrict any html code .
[check for this page](http://materializecss.com/forms.html#select)

**Review by amanoese — CHANGES_REQUESTED** [review]

**Comment by ukenhk-kuni077** [comment]
I pushed new my code.
Please check it out.
Thanks.

**Pull request closed without merging**